dept44-starter/src/main/resources/config/application.properties

```diff
@@ -34,7 +34,7 @@ spring.jackson.deserialization.fail-on-unknown-properties=false
 spring.jackson.deserialization.fail-on-null-for-primitives=false
 
 # Enable spring to read properties from local .env file
-spring.config.import=optional:file:.env[.properties]
```

Comment:
Det är väl bara prefixet som ska tas bort - betyder inte det här att vi inte längre försöker läsa in `.env`-filer?

Reply:
Men det är väl det som gjorts?

Reply:
Jag har verifierat lokalt med snapshot version och det fungerar att använda .env

Reply:
Jamen dåså 👍